apps/publishers/admin.py

```
@@ -9,7 +9,7 @@
 class PublisherMemberInline(admin.TabularInline):
     model = PublisherMember
     extra = 0
-    fields = ["user", "role"]
+    fields = ["user", "group"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)